news/18.bugfix

```diff
@@ -0,0 +1,2 @@
+Fix highlight of current item in nav for image and file.
```

Comment:
For future updates, this will probably better be something like:
``Updated Mockup to version 2.3.4.``
Also because normally several changes are in a Mockup release.
But that would need a Mockup release on npm first.